### PR TITLE
Mazerator 0.0.2

### PR DESCRIPTION
**Pull request by AwesomeKalin**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/67363)

### CODE REVIEW

**Comment by yao-msft** [comment]
Hi @AwesomeKalin ,
Is it possible to use release links from https://github.com/MazeratorGame/Mazerator/releases for InstallerUrl? That will pass the Validation-Domain check.
Thanks

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/67363#issuecomment-1194500457

File: manifests/w/WaterbugStudios/Mazerator/0.0.2/WaterbugStudios.Mazerator.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WaterbugStudios.Mazerator
+PackageVersion: 0.0.2
+PackageLocale: en-US
+Publisher: Waterbug Studios
```
Comment (ItzLevvie) [inline manifests/w/WaterbugStudios/Mazerator/0.0.2/WaterbugStudios.Mazerator.locale.en-US.yaml:7]:
```suggestion
Publisher: Waterbug Studios/Awesome Arcade
```
cc @AwesomeKalin: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by another PR;

**Pull request closed without merging**